src/components/SelectableCard/SelectableCard.scss

```diff
@@ -0,0 +1,32 @@
+@use '../../variables.scss';
+
+$block: '.#{variables.$ns}selectable-card';
+
+#{$block} {
+    position: relative;
+    display: flex;
+    padding: 26px;
+    height: 80px;
+    align-items: center;
+    justify-content: center;
+    cursor: pointer;
+
+    &__icon {
+        position: absolute;
+        top: 4px;
+        right: 4px;
+        color: var(--g-color-base-brand);
+    }
+
+    &__fake-button {
+        width: 69px;
+        height: 28px;
+        background-color: var(--g-color-base-brand);
+        display: flex;
+        align-items: center;
+        justify-content: center;
+        user-select: none;
+        user-select: none;
+        user-select: none;
```

Comment:
`user-select: none;` 3 times?

Reply:
I've fixed in in radius tab PR's: https://github.com/gravity-ui/landing/pull/220/files#diff-42177eb1d95fa6db32580d2f3505f57da1a20e8239dcc2b81ae087dce5a5b009L29